conda_concourse_ci/execute.py

```diff
@@ -196,6 +196,7 @@ def get_build_task(
         use_staging_channel=False,
         automated_pipeline=False,
         pull_recipes_resource=None,
+        sign_artifacts=True,
```

Comment:
```suggestion
```

I intended to make this a flag, but I think I'll leave it in as mandatory behavior.